index.test.ts

```diff
@@ -5,11 +5,12 @@ import path from 'path';
 import { promisify } from 'util';
 import _treeKill from 'tree-kill';
 import retry from 'async-retry';
+import { randomBytes } from 'crypto';
 
 const treeKill = promisify(_treeKill);
 
-process.env.SESSION_SECRET =
-  'HbGUBzUcVC4ghjg4w4T2Dz4z7dByYCz7GTAUDwaUEEFc2WxkjuPMyqnTtZ4H3hMp';
+// this is passed to Keystone
+process.env.SESSION_SECRET = randomBytes(32).toString('hex')
```

Comment:
We should probably only do this for the `production` run-up, if we want to verify that our defaults are working